packages/flame/lib/src/components/component.dart

```diff
@@ -44,6 +44,17 @@ class Component {
   Component? get parent => _parent;
   Component? _parent;
 
+  /// Returns the first child that matches the given type [T].
+  T? child<T extends Component>() {
```

Comment:
I think the name is a bit unclear, but I think the method will be very useful!

Reply:
Any suggestions? My first idea was `get`, but I thought it was too generic, `child` I think kind of goes well since with have an `children`.

My other ideas were:
 - findChild
 - getChild
 - fetchChild
 - firstChild

Reply:
Yeah, I need a simple and safe way to recover an specific child of a component, doing `children.whereType<Bla>().first`, even though is not that long, starts to became repetitive after a couple of places that you write it and I feel that it isn't very much safe as it will throw and error is it can't find the child, so by returning null it makes that a little bit easier too as it is explicit that the child may not exist

Reply:
Maybe `firstChild` could be a good name?

Reply:
I like `firstChild` over `findChild` because it kind makes it more clear what this method does

Reply:
Lol, `firstChild` was even in your list of suggestions, missed that 😅